doc/table_of_contents.md

```diff
@@ -9,7 +9,7 @@
 - [blocks_and_headers](chain/blocks_and_headers.md) - How Grin tracks blocks and headers on the chain
 - [contractideas](contractideas.md) - Ideas on how to implement contracts
 - [dandelion/dandelion](dandelion/dandelion.md) - About transaction propagation and cut-through. Stemming and fluffing!
-- [dandelion/simulating](dandelion/simulating.md) - Dandelium simulation - aggregating transaction without lock_height Stemming and fluffing!
+- [dandelion/simulation](dandelion/simulation.md) - Dandelium simulation - aggregating transaction without lock_height Stemming and fluffing!
```

Comment:
Could you fix the following typo:
It's not ```Dandelium simulation``` it's ```Dandelion simulation```

Reply:
done!